src/everwillow/statelib/state.py

```
@@ -0,0 +1,624 @@
+
+    if TYPE_CHECKING:
+        _mapping: TypingChainMap[KeyPath, V]
+        _segments: dict[object, SegmentRecord[V]]
+        _segment_order: list[object]
+        _primary_segment: object
+
+    def __new__(cls, *args, **kwargs):
+        del args, kwargs  # unused
+        raise TypeError(
+            "'FlatState' should never be directly instantiated, use 'FlatState.from_pytree' instead"
+        )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)